build/agent/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nginx:1.22.1-alpine
```

Comment:
Can `nginx:1.22.1-alpine` changes behind the scenes so that its alpine version is not the same as we try configure?

Reply:
I don't think so... @lucacome do you know?